src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -300,7 +289,6 @@ public void widgetDefaultSelected(SelectionEvent e)
                 defaultActionRequested((TreeItem) item);
             }
 
-            @Override
             public void widgetSelected(SelectionEvent e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)